src/app/common/interfaces/IUser.ts

```diff
@@ -0,0 +1,9 @@
+/* For features */
+export class IUser{
```

Comment:
Is it class or an interface? :) 
You say it's a class, but name says that it's interface: **I**User

Btw, let's use interfaces instead of classes for data models